rust-agent/.ci/install_go.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
# Location

I couldn't attend the meeting yesterday so don't know exactly what was discussed. However, if we're going to start migrating the main components into this repo, I think it makes sense to create some sort of top-level directory which the components willl live under (`./components/{agent,shim,proxy,runtime}` or `./src/{agent,shim,proxy,runtime}` or similar).

# Name

Obviously, the new agent is written in rust, but I don't see why the directory name needs to include `rust` :smile: There is no other agent in this repo so why can't we just call it `agent` (or as mentioned above, `${dir}/agent/`)?

Reply:
We didn't have a decision on the directory structure. On the other hand, we decide to have a PR firstly, then we expect we may have more inputs.

Reply:
Thanks @gnawux.

Reply:
Guess we should settle and agree on a dir structure for this repo then before we land the first major PR....
I'm presuming that we are indeed intending to (gradually, piecemeal) move all components of kata into this one repo, as we know having the multiple repos has ended up causing us too much pain (particularly around complex multi-item PRs and managing the CI).

Thoughts then:
1) I'd be in favour of placing the various items under a slightly deeper tree. For a start, maybe a top level `src` dir, to place source based items under, so we'd have `./src/agent` for this code.
1) I can see we may not need `-rust` in the name of this item. I'd suggest either just `agent`, or if we think that might confuse things with the existing agent repo, then maybe just `agent2` ?
1) I see we have a `.ci` subdir in this PR/item. I strongly suspect we should have a single top level CI dir in this whole repo that can cover all sub-items (agent, runtime etc. in the future). What do you think @chavafg ?
1) and.... maybe we can just call that CI dir `ci` rather than `.ci` - no need to hide it any more, and it can and probably should become a first class citizen...
1) I guess in which case, maybe it would be `./tools/ci` in the hierachy, rather than just `ci` at the top level?

make sense? Any objections ?

Reply:
yeah, for CI make sense to to have a top `ci` directory instead of having one per component.